.asf.yaml

```diff
@@ -0,0 +1,44 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+github:
+  description: EventMesh is a dynamic cloud-native eventing infrastruture used to decouple the application and backend middleware layer, which supports a wide range of use cases that encompass complex multi-cloud, widely distributed topologies using diverse technology stacks.
+  homepage: https://eventmesh.apache.org/
+  labels:
+    - event-mesh
+    - event-gateway
+    - event-driven
+    - event-streaming
+    - event-sourcing
+    - event-governance
+    - cloud-native
+    - serverless
+    - workflow-serverless
+    - esb
+    - message-bus
+    - CQRS
```

Comment:
I remember capital characters are not allowed in the labels.

Reply:
I think it is ok.

Reply:
> I think it is ok.

By “not allowed”, I mean it causes the settings failed to take effect